crates/oci-distribution/Cargo.toml

```diff
@@ -16,7 +16,7 @@ edition = "2018"
 reqwest = { version = "0.10.4", features = ["json"] }
 failure = "0.1"
 tokio = "0.2.13"
-serde = "1.0"
+serde = { version = "1.0", features = ["derive"] }
```

Comment:
Still trying to learn about how features work... does this mean "the feature `derive` is added" or "all the features except `derive` are disabled"?

Reply:
Features are generally additive (i.e., they _should_ be additive but not every crate follows that recommended pattern). So putting the `derive` feature here adds additional functionality (namely the ability to derive `Serialize` and `Deserialize`). You may see `default-features = false` which turns off all features that are marked as default in that dependency's Cargo.toml file. You can then add features on top of that.